src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,15 @@
-export const CommentInfo = () => <>Put the comment here</>;
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
```

Comment:
You could use destructuring, for example:  `const { name, email, body } = comment;` and then use only `name`, `email`, `body`  instead `coment.name` etc.

Reply:
> You could use destructuring, for example: `const { name, email, body } = comment;` and then use only `name`, `email`, `body` instead `coment.name` etc.

Good point